multi-agent/tests/e2e/test_session_stress.py

```
@@ -265,6 +300,72 @@ def get_session_state(self, session_id: str) -> Dict:
+        elif llm_type == "openai":
+            config = {
+                "type": "openai",
+                "model_name": model_name,
+                "api_key": api_key,
+                "base_url": base_url or "https://generativelanguage.googleapis.com/v1beta/openai",
+                "verify_ssl": True,
+            }
+        else:
+            raise ValueError(
+                f"Unsupported llm_type '{llm_type}'. Use 'google_genai' or 'openai'."
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)